HLTrigger/Configuration/python/Eras.py

```diff
@@ -0,0 +1,15 @@
+import FWCore.ParameterSet.Config as cms
+
+# import the relevant eras from Configuration.Eras.*
+from Configuration.Eras.Modifier_trackingPhase1_cff import trackingPhase1
+
+def modifyHLTforEras(fragment):
+    """load all Eras-based customisations for the HLT configuration"""
+
+    # modify the HLT configuration for the Phase I pixel geometry
+    from HLTrigger.Configuration.customizeHLTTrackingForPhaseI2017 import customizeHLTPhaseIPixelGeom
+    trackingPhase1.toModify(fragment, customizeHLTPhaseIPixelGeom)
+
+    # modify the HLT configuration to run the Phase I tracking in the particle flow sequence
+    from HLTrigger.Configuration.customizeHLTTrackingForPhaseI2017 import customizeHLTForPFTrackingPhaseI2017
+    trackingPhase1.toModify(fragment, customizeHLTForPFTrackingPhaseI2017)
```

Comment:
I think we want to have a single tracking configuration for the HLT, `phase1Pixel` makes the most sense.
What are (for the offline tracking) the use cases for `Run2_2017` vs `Run2_2017_trackingRun2` vs `Run2_2017_trackingPhase1PU70` ?
I'm asking in case it's something we should try to support on the HLT side as well.